dev/TabView/InteractionTests/TabViewTests.cs

```diff
@@ -140,9 +141,9 @@ public void CloseSelectionTest()
                 Verify.AreEqual(selectedIndexTextBlock.DocumentText, "3");
 
                 Log.Comment("When the selected tab is last and is closed, selection should move to the previous item.");
-                closeButton = FindCloseButton(lastTab);
-                Verify.IsNotNull(closeButton);
-                closeButton.InvokeAndWait();
+
+                // Use Middle Click to close the tab:
+                lastTab.Click(PointerButtons.Middle);
```

Comment:
Does this Click method do a WaitForIdle?

Reply:
Makes sense to add WaitForIdle here.